applications/GeoMechanicsApplication/custom_conditions/Pw_normal_flux_condition.hpp

```diff
@@ -102,7 +104,8 @@ class KRATOS_API(GEO_MECHANICS_APPLICATION) PwNormalFluxCondition : public PwCon
     {
         KRATOS_SERIALIZE_LOAD_BASE_CLASS(rSerializer, Condition)
     }
+};
 
-}; // class PwNormalFluxCondition.
+// class PwNormalFluxCondition.
```

Comment:
This layout seems a bit strange

Reply:
Good catch! For some unclear reason, the formatting has been changed here. I have reverted this change and the other ones that were similar.